src/views/CollectionIndex/Page.tsx

```diff
@@ -0,0 +1,63 @@
+import topArrow from "images/arrow-down.svg";
+import bottomArrow from "images/arrow-up.svg";
+import React, { useState } from "react";
+import { Link } from 'react-router-dom';
+import { PageHeader } from "../../components/Header/PageHeader";
+
+
+const Page = props => {
+  const [show, showOverlay] = useState(false);
+  const { data, history } = props;
+  const toShow = localStorage.getItem('show')
+  if(toShow === "true") {
+    showOverlay(true)
+  }
+  localStorage.removeItem('show')
+
```

Comment:
So, if you use localStorage. Use the hook instead. I think it's called `useLocalStorage`

But you don't need to use localStorage for the overlay right? You can just `useState`, because it's not important to persist this state. Please correct me if I'm wrong.